app/models/project.rb

```diff
@@ -12,8 +12,9 @@
 class Project < ActiveRecord::Base
   has_many :configs
   has_many :metric_samples
-
   has_and_belongs_to_many :users
+  has_many :ownerships
```

Comment:
@DrakeW I'm not sure that you need both - have a look at http://guides.rubyonrails.org/association_basics.html#choosing-between-has-many-through-and-has-and-belongs-to-many isn't it a question of using one or the other?  also, are all these relations needed?  if it was me I would experimentally delete some and see if tests failed.

As a general rule, only add the absolute minimum code required to get tests to pass.  Try to avoid adding extra code because you think it will be needed in the future or it "makes sense"